app/views/admin/judges/_form.html.erb

```diff
@@ -16,11 +16,18 @@
         <%= f.text_field :last_name, class: 'form-control' %>
       </div>
     </div>
+
+    <div class='row form-group'>
```

Comment:
Adding the checkbox in this form is such a great idea, thank you!  And an upside of using the 'active' scope explicitly rather than as a default scope is that we could include all Judges in this table and this checkbox could be 'checked' to 'hide' and 'unchecked' to 'unhide'.  And that should just work with the code you have!